torch/_functorch/partitioners.py

```diff
@@ -84,13 +84,17 @@ def _extract_graph_with_inputs_outputs(joint_graph, inputs, outputs):
 
 
 def _is_primal(node):
-    return node.op == "placeholder" and "tangents" not in node.target
+    return node.op == "placeholder" and "tangents" not in node.target and not _is_bwd_seed_offset(node)
 
 
 def _is_tangent(node):
     return node.op == "placeholder" and "tangents" in node.target
 
 
+def _is_bwd_seed_offset(node):
+    return node.op == "placeholder" and ("bwd_seed" in node.target or "bwd_base_offset" in node.target)
```

Comment:
How do you know that you are guaranteed to be able to get these names?

Reply:
This is because of this - https://github.com/pytorch/pytorch/pull/97377/files#diff-df954bbf954d2dcb81f687876053267ffa4ddb36ed86b7d2bd76319ff2b94416R2317

This problem is similar to the naming requirements of `primals` and `tangents` for partitioner.